package.json

```diff
@@ -320,44 +336,44 @@
   },
   "devDependencies": {
     "@types/mocha": "10.0.1",
-    "@types/node": "20.8.4",
-    "@types/react": "18.3.3",
+    "@types/node": "22.5.0",
+    "@types/react": "18.3.4",
     "@types/react-dom": "18.3.0",
     "@types/react-select": "5.0.1",
     "@types/sinon": "17.0.3",
-    "@typescript-eslint/eslint-plugin": "6.9.0",
-    "@typescript-eslint/parser": "6.9.0",
+    "@typescript-eslint/eslint-plugin": "8.3.0",
+    "@typescript-eslint/parser": "8.3.0",
     "@vscode/test-cli": "0.0.4",
     "@vscode/test-electron": "2.3.9",
     "assert": "2.1.0",
     "console-browserify": "1.2.0",
     "cross-env": "7.0.3",
     "crypto-browserify": "3.12.0",
-    "css-loader": "4.2.0",
-    "eslint": "8.52.0",
-    "eslint-config-prettier": "9.0.0",
-    "eslint-plugin-import": "2.29.0",
-    "eslint-plugin-prettier": "5.0.1",
-    "eslint-plugin-react": "7.33.2",
-    "eslint-plugin-react-hooks": "4.6.0",
+    "css-loader": "7.1.2",
+    "eslint": "8.57.0",
+    "eslint-config-prettier": "9.1.0",
+    "eslint-plugin-import": "2.29.1",
+    "eslint-plugin-prettier": "5.2.1",
+    "eslint-plugin-react": "7.35.0",
+    "eslint-plugin-react-hooks": "4.6.2",
     "license-check-and-add": "4.0.5",
     "maven": "5.0.0",
     "npm-run-all": "4.1.5",
-    "postcss": "8.4.31",
+    "postcss": "8.4.41",
     "postcss-scss": "4.0.9",
     "prettier": "3.3.3",
     "process": "0.11.10",
-    "sass": "1.70.0",
+    "sass": "1.77.8",
     "sass-loader": "14.0.0",
     "sinon": "17.0.1",
     "stream-browserify": "3.0.0",
     "style-loader": "1.2.1",
-    "stylelint": "15.11.0",
-    "stylelint-config-standard": "34.0.0",
-    "stylelint-scss": "5.2.1",
+    "stylelint": "16.8.2",
+    "stylelint-config-standard": "36.0.1",
+    "stylelint-scss": "6.5.1",
     "ts-jest": "29.1.2",
     "ts-loader": "9.5.1",
-    "typescript": "5.2.2",
+    "typescript": "5.5.4",
```

Comment:
why is this downgraded?

Reply:
It's not downgraded. I updated it from 5.2.2 to 5.5.4 to match the version in legend-studio.

Reply:
sorry. my bad